src/EduFramework/Commands/StartCommand.php

```diff
@@ -61,7 +89,8 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             'Pour arrêter le serveur de développement, appuyer sur CTRL+X CTRL+C'
         );
 
-        exec("composer edu:start");
+        exec("Composer\\Config::disableProcessTimeout",);
```

Comment:
@CapelleGab, la commande "sh: ComposerConfig::disableProcessTimeout: command not found"

Reply:
Je l'ai supprimé